### DISCUSSION TITLE
Adding Underline Text in finplot Library

### DISCUSSION THREAD

**Discussion by NijanJ** [highfestiva/finplot#430] [Q&A]
Hi,
 
How to incorporate underlined text within my visualizations. I have thoroughly explored the library's documentation and examples, but I have not yet found a clear solution for underlining text using Finplot.

**Comment by highfestiva** [comment]
Hi,

Right you are. There is one quite vague example, snp500.py. But added a proper `labels()` call example in `pandas-df-plot.py` in commit f77ccfe.
